.circleci/config.yml

```diff
@@ -97,6 +96,10 @@ yaml-templates:
       paths:
         - dist
 
+  images_environment: &images_environment
+    CI_SERVICE_CONTROLLER_IMAGE: quay.io/nicob87/service-controller
+    CI_SITE_CONTROLLER_IMAGE: quay.io/nicob87/site-controller
```

Comment:
@nicob87 can we use some quay.io organization instead of a private repo?

Reply:
your question is related to grs question ... I am using my private just for showing how it works ... my initial intention was to the, change it to quay.io/skupper ...